spot-client/src/common/backend/utils.js

```diff
@@ -229,6 +229,44 @@ export function getRemotePairingCode(serviceEndpointUrl, jwt) {
     });
 }
 
+/**
+ * Checks if it's ok to call given phone number.
+ *
+ * @param {string} serviceEndpointUrl - The URL pointing to the phone authorize service.
+ * @param {string} phoneNumber - E.164 formatted phone number to be checked.
+ * @returns {Promise<{allow: boolean, message: string}>}
```

Comment:
I think this jsdoc is incorrect. I think the promise returns nothing on resolve.

Reply:
oh yes it got outdated at some point when simplifying